tests/unit-tests/InAppHelperTests.swift

```diff
@@ -174,5 +174,9 @@ class InAppHelperTests: XCTestCase {
         func getRemoteConfiguration() -> Pending<RemoteConfiguration, SendRequestError> {
             fatalError()
         }
+        
+        func getEmbeddedMessages() -> IterableSDK.Pending<IterableSDK.SendRequestValue, IterableSDK.SendRequestError> {
+            fatalError()
```

Comment:
Why is this mock function calling IterableSDK.Pending but the other mock functions are not?

Reply:
Not sure, this was automatically generated. It's probably just extra clear namespacing from Xcode.